addons/web/static/src/views/relational_model.js

```diff
@@ -231,27 +229,10 @@ class RequestBatcherORM extends ORM {
     /**
      * @override
      */
-    async webSearchRead(/*model*/) {
```

Comment:
@aab-odoo I think that the previous implementation of `webSearchRead` wasn't doing anything?
Just adding in a list and popping but still doing the super call every single time

Reply:
I am adding the domain in the key